engine/src/main/java/org/terasology/rendering/dag/nodes/AlphaRejectBlocksNode.java

```diff
@@ -52,7 +52,7 @@
  * the color stored in the frame buffer and the resulting color overwrites the previously stored one.
  */
 public class AlphaRejectBlocksNode extends AbstractNode implements WireframeCapable {
-    private static final ResourceUrn CHUNK_SHADER = new ResourceUrn("engine:prog.chunk");
```

Comment:
1. These are changes that are not relevant to the goal of this PR. It's ok to do them in this PR if they caught your attention but they should be in their own commit.
2. this URN, when used with getMaterial, returns a material, not a shader.